junit-jupiter-api/src/main/java/org/junit/jupiter/api/IndicativeSentencesGeneration.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.api;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apiguardian.api.API;
+
+/**
+ * {@code @IndicativeSentencesGeneration} is used to declare a custom parameters
+ * by {@code IndicativeSentences}, if this notation has some not declared
+ * parameters, it will use the default values instead.
+ *
+ * @since 5.7
+ * @see DisplayName
+ * @see DisplayNameGenerator
+ */
+@DisplayNameGeneration(DisplayNameGenerator.IndicativeSentences.class)
+@Target({ ElementType.ANNOTATION_TYPE, ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Inherited
+@API(status = EXPERIMENTAL, since = "5.7")
+public @interface IndicativeSentencesGeneration {
+
+	String DEFAULT_SEPARATOR = ", ";
+	Class<? extends DisplayNameGenerator> DEFAULT_GENERATOR = DisplayNameGenerator.Standard.class;
+
+	/**
+	 * Custom separator for indicative sentences generator.
+	 *
+	 * @return custom separator for indicative sentences
+	 */
+	String separator() default "";
```

Comment:
Why don't we default to `", "` here?

Reply:
It seems to be done [here](https://github.com/Kazhuru/junit5/blob/0c98ec8d3e030062020ca6550e59274fdf338309/junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java#L239) anyway

Reply:
I love how you hit the nail on the head with ideas that I thought for a while 🤓  with no so transparent decisions, I left "" as default because the recommendations on this comment:
https://github.com/junit-team/junit5/pull/2124#discussion_r359873426.
But yeah as you say in that section im checking for an empty string of separator and then using the default static variable.
Anyway I can change that quick for sure if thats better.

Reply:
Ahah, telepathy? xD
I think the initial remark was more about the fact that we need to set a default value, not necessarily about the value itself
@marcphilipp what do you think?

Reply:
I think I was thinking about the generator being used without the annotation. If we require the use of the annotation using the default value of the annotation attribute is fine with me.

Reply:
Oh I see now anyway this default value was not used because it later is replaced, roger on that, then to set the value as default here! 👍